### PR TITLE
fix(editor): Restore V1 keybinding, Space Key to toggle panning

### PR DESCRIPTION
**Pull request by r00gm**
## Review / Merge checklist

- [ ] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/editor-ui/src/components/canvas/Canvas.vue [hunk 1]
```diff
@@ -122,7 +122,7 @@ const classes = computed(() => ({
 
 const disableKeyBindings = computed(() => !props.keyBindings);
 
-const panningKeyCode = 'Shift';
+const panningKeyCode = ' ';
```
Comment (r00gm) [inline packages/editor-ui/src/components/canvas/Canvas.vue:125]: https://developer.mozilla.org/en-US/docs/Web/API/UI_Events/Keyboard_event_key_values#whitespace_keys

**Review by cstuncsik — APPROVED** [review]

**Review by cstuncsik — DISMISSED** [review]

File: packages/editor-ui/src/components/canvas/Canvas.vue (hunk 1 above)
Comment (tomi) [reply packages/editor-ui/src/components/canvas/Canvas.vue:125]: 👍 Could even add it as a comment to the code

**Review by tomi — DISMISSED** [review]
🥳

File: packages/editor-ui/src/components/canvas/Canvas.vue (hunk 1 above)
Comment (r00gm) [reply packages/editor-ui/src/components/canvas/Canvas.vue:128]: done

**Review by tomi — APPROVED** [review]
🚀

**Comment by janober** [comment]
Got released with [`n8n@1.60.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.60.0)

**Pull request merged**